build.sbt

```diff
@@ -70,10 +68,12 @@ val coreDeps = Seq(
   "org.typelevel" %% "munit-cats-effect" % munitCatsEffectV
 )).map(_ % Test)
 
+ThisBuild / resolvers ++= Resolver.sonatypeOssRepos("snapshots")
+
 val scala213 = "2.13.10"
 ThisBuild / crossScalaVersions := Seq("2.12.17", scala213, "3.2.1")
 ThisBuild / scalaVersion := scala213
-ThisBuild / tlBaseVersion := "0.8"
+ThisBuild / tlBaseVersion := "0.9"
```

Comment:
Because we are dropping the `reactive-streams` dependencies, Ross would tell us to bump the base version. But we are not actually breaking any APIs, only changing the internal implementations.

Reply:
Your transitive dependencies are your API in a binary-distributed ecosystem in a broken society that rejects the good hygiene of sbt-explicit-dependencies.  I don't make the rules.